tests/integration_test/transcoding_print_options/transcoding_print_options_test.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2019 Google LLC
```

Comment:
what is the plan for re-organize these tests?

Maybe just put into grpc_test/grpc_integration_test.go

Reply:
working on it.

moved to grpc_transcoding_test